### ISSUE TITLE
Box is too small for the grid

### ISSUE THREAD

**Issue by Silence2107** [smash-transport/smash#48]
Dear maintainers,
I have faced an error message in SMASH-2.2 that I did not expect.

Given that I did not need any specific potentials enabled, I followed instruction on http://theory.gsi.de/~smash/userguide/2.2/input_general_.html regarding delta time, and set time_step_mode to none. It worked fine for Au+Au simulation in SMASH-1.8 and SMASH-2.2.

However, when I tried same approach in the infinite matter (box) simulation, I got rather different behaviour. SMASH-1.8 ate same instruction just fine (at least so it looks), while SMASH-2.2 threw an error

```
SMASH failed with the following error:
Input error: Your box is too small for the grid.
The minimal length of the box is given by: 20.000000 fm with the given timestep size.
If you have large timesteps please reduce them.
A larger box or the use of testparticles also helps.
Please take a look at your config.
```

This confuses me, since I haven't explicitly declared any potential. Here is my config:

```
Version: 1.8 # minimal SMASH version to use with this config file

Logging:
    default: INFO

General:
    Modus:          Box
    Time_Step_Mode: None
#    Delta_Time:     0.1
    End_Time:       1000.0
    Randomseed:     342002906
    Nevents:        25

Output:
#    Output_Interval: 1.0
    Output_Times: [700.0]
    Particles:
        Only_Final: No
        Format:          ["Root"]

Modi:
    Box:
        Length: 10
        Initial_Condition: "peaked momenta"
        Start_Time: 0.0
        Temperature: 0.5661475236840086
        Init_Multiplicities:
            3122: 0
            2112: 593
```

I can clearly tell it argues about this time_step_mode, however if it shouldn't be none, should I be concerned regarding my other results? I tried to look into changelogs between 1.8 and 2.2, but I am not as good in the lookup. Could you please point out what does it mean and what should I do to 'fix' my simulation?

Please don't hurry, holidays are more important.

Thanks for your time,
Pavlo Panasiuk

**Comment by gabriele-inghirami** [comment]
Dear Pavlo,

as you already guessed, in this period several members of our group are on vacation, including those more expert about box simulations.
We will come back to this issue as soon as possible, but probably not before the end of the holidays.

Thank you for your patience,
Gabriele

**Comment by justinmohs** [comment]
While we look into this issue a bit more in detail, I would suggest you to perform box calculations with a fixed time step @Silence2107. The reason is that finding the interactions between two particles on opposite ends of the box (that should be possible due to the periodic boundaries) is a bit more difficult. In order to do so, we split our box into cells which have to be larger than the maximum interaction range of a particle traveling at the speed of light throughout a time step. This is of course not feasible in the timestepless mode, since we just perform one large time step for the entire evolution effectively.
If the cells would be required to be larger than the box, we throw the error you reported. 

In short: 
* I would rerun the box calculations setting a fixed time step of ~ 1 fm
* I think the results you obtained with smash 1.8 without a time step are probably inaccurate
* You results in the collider mode should be reliable

**Comment by AxelKrypton** [comment]
@Silence2107 We have opened a private issue within our team to have a closer look to this issue and we agree that from the user perspective there should be a more explanatory and helpful error message and, possibly, a better check of the used parameters in the code. For sure we'll add some better description to the user guide, which will be likely shipped within next release. Thanks for bringing this to our attention. Since we track this aspect elsewhere, I will close the issue for now.
